src/main/webapp/app/entities/notification.model.ts

```diff
@@ -94,5 +95,23 @@ export const TUTORIAL_GROUP_ASSIGNED_TITLE = 'You have been assigned to lead a t
 
 export const TUTORIAL_GROUP_UNASSIGNED_TITLE = 'You have been unassigned from leading a tutorial group';
 
+export const NEW_MESSAGE_TITLE = 'New message';
```

Comment:
why do we need those here? Are these not strings that you would put into translation files? (i.e. json)

Reply:
This will be changed after i adapt the translation PR.